rust/core-lib/src/view.rs

```diff
@@ -351,6 +352,20 @@ impl View {
         self.set_selection(text, new_sel);
     }
 
+    pub fn page_up_down(&mut self, movement: Movement) {
+        let first_line: i64 = self.first_line as i64;
+        let scroll_height: i64 = self.scroll_height() as i64;
+        match movement {
+            Movement::DownPage => {
+                self.set_scroll(first_line - scroll_height, first_line);
```

Comment:
Right now this only updates `first_line` and `height` in `View` but does not tell the frontend to actually scroll to that position. So for now nothing happens. 
To make it scroll you have to set `self.scroll_to = Some(<position>);`
There are some functions in `View` where this is already used, for example in `scroll_to_cursor`, which might be helpful.